src/functions/template-tags/organizer.php

```diff
@@ -253,34 +253,64 @@ function tribe_get_organizer_email( $postId = null, $antispambot = true ) {
 	 *
 	 * @param int  $postId    Can supply either event id or organizer id, if none specified, current post is used
 	 * @param bool $full_link If true outputs a complete HTML <a> link, otherwise only the URL is output
-	 * @param bool $echo      If true, echo the link, otherwise return
+	 * @param bool $echo      Deprecated. If true, echo the link, otherwise return
 	 *
 	 * @return string Organizer Name and Url
 	 */
 	function tribe_get_organizer_link( $postId = null, $full_link = true, $echo = false ) {
 
 		// As of TEC 4.0 this argument is deprecated
 		// If needed precede the call to this function with echo
-		if ( $echo != false ) _deprecated_argument( __FUNCTION__, '4.0' );
+		if ( $echo != false ) _deprecated_argument( __FUNCTION__, '4.0', 'As of TEC 4.0 this argument is deprecated. If needed, precede the call to this function with echo' );
 
 		$org_id = tribe_get_organizer_id( $postId );
 		if ( class_exists( 'Tribe__Events__Pro__Main' ) && get_post_status( $org_id ) == 'publish' ) {
-			$url = esc_url_raw( get_permalink( $org_id ) );
+			/**
+			 * Filter the organizer link target attribute.
+			 *
+			 * @since TBD
+			 *
+			 * @param string the target attribute string. Defaults to "_self".
+			 */
+			$target = apply_filters( 'tribe_get_event_organizer_link_target', '_self' );
+			$rel    = ( '_blank' === $target ) ? 'noopener noreferrer' : '';
+			$url    = esc_url_raw( get_permalink( $org_id ) );
+
 			if ( $full_link ) {
 				$name = tribe_get_organizer( $org_id );
-				$attr_title = the_title_attribute( array( 'post' => $org_id, 'echo' => false ) );
-				$link = ! empty( $url ) && ! empty( $name ) ? '<a href="' . esc_url( $url ) . '" title="'.$attr_title.'">' . $name . '</a>' : false;
+
+				if ( empty( $url ) || empty( $name ) ) {
+					$link = false;
+				} else {
+					$link = sprintf(
+						'<a href="%s" title="%s" target="%s" rel="%s">%s</a>',
+						esc_url( $url ),
+						the_title_attribute(
+							[
+								'post' => $org_id,
+								'echo' => false,
+							]
+						),
+						esc_attr( $target ),
+						esc_attr( $rel ),
+						esc_html( $name )
+					);
+				}
 			} else {
 				$link = $url;
 			}
 
-			// Remove this in or before 5.x to fully deprecate the echo arg
-			if ( $echo ) {
-				echo apply_filters( 'tribe_get_organizer_link', $link, $postId, $echo, $url );
-			} else {
-				return apply_filters( 'tribe_get_organizer_link', $link, $postId, $full_link, $url );
+			/**
+			 * Filter the organizer link HTML
+			 *
+			 * @since 4.0
+			 *
+			 * @param string the link HTML.
+			 */
+			return apply_filters( 'tribe_get_organizer_link', $link, $postId, $full_link, $url );
```

Comment:
:no_entry_sign: Variable "$postId" is not in valid snake_case format, try "$post_id"

<sub>hash: e15475b69f49b995bbf10e075a79d8f6</sub>